### ISSUE TITLE
Remove statements about recent split

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4170]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4170|
|Type|defect|
|Created|2009-12-15 01:18:09Z|

The following files state that there was a recent split, but it looks like this happened (4) years ago:

 - [twisted/mail/topfiles/README](twisted/mail/topfiles/README)
 - [twisted/pair/topfiles/README](twisted/pair/topfiles/README)

Attachments:

* [split-topfiles.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ccc/cccdb77810cde111644e15fb38ed3266fcae8d38/4f1e8f0fed3e68e0751139bac3b2d628f7663bb2.patch) (776 bytes) - added by thijs on 2009-12-15 01:18:26Z - 
* [split-topfiles.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/ccc/cccdb77810cde111644e15fb38ed3266fcae8d38/3e9333bfda9f693ab0522ac23318e8404e166845.patch) (636 bytes) - added by thijs on 2009-12-18 04:58:55Z - patch with newlines

<details><summary>Searchable metadata</summary>

```
trac-id__4170 4170
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__Twisted_10_0 Twisted-10.0
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1260839889000000 1260839889000000
changetime__1261112558000000 1261112558000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

There aren't newlines on either of the files anymore if you apply the patch - not sure if that's desirable. Other than that, apply away.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [jesstess](#note_2):
> There aren't newlines on either of the files anymore if you apply the patch - not sure if that's desirable. Other than that, apply away.

Thanks for the review. I attached a new patch with newlines that I'll apply now.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27816]) Apply split-topfiles.2.patch: remove statements about recent Twisted split.

Author: thijs
Reviewer: jesstess
Fixes: [#4170](https://github.com/twisted/twisted/issues/4170)
